src/types/actions.ts

```diff
@@ -129,6 +129,11 @@ export enum ColonyAndExtensionsEvents {
   AgreementSigned = 'AgreementSigned',
 }
 
+export type ActionUserRoles = {
+  id: ColonyRole;
+  setTo: boolean;
+};
+
```

Comment:
Could you clean up the imports in this file? They still contain the old `~core` or `~data`